ispdb/mnet.ne.jp.xml

```diff
@@ -4,18 +4,28 @@
     <domain>mnet.ne.jp</domain>
     <displayName>Mnet メール サービス</displayName>
     <displayShortName>Mnetメール</displayShortName>
+    <incomingServer type="imap">
+      <hostname>mail.mnet.ne.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
```

Comment:
Thanks for adding the SSL configs. I was able to connect to the server (but not login, because I don't have an account).

I noticed that `STARTTLS` with IMAP on port 143 and POP3 on port 110 also work. Could you please add those configs as well, as second config? The SSL config is preferred, and should be the first (so the config you created was great), but we capture all secure and working configs, so it would be nice to also capture STARTTLS as lesser preferred alternative.

Reply:
I don't have an account to test. But connections to IMAP/143 and POP3/110 using STARTTLS work fine, so I added entries for them in 295deaaeea2e93504a958da2b4000a62dad91726.